src/views/Terms/__tests__/Term.test.js

```diff
@@ -21,6 +21,7 @@ describe('Term component rendered with English', () => {
     const searchBoxTitle = "Search NCI's Dictionary of Cancer Terms";
     // Get first result from mock file A.json
     const term = getFixture(`${fixturePath}/${language}/${queryFile}`).results[0];
+    const idOrName = term.prettyUrlName ? term.prettyUrlName : term.termId;
```

Comment:
This test should be using a static constant. I see it is just a tweak of what was there before, but this test is not clear enough in what is being tested. Technically since only one item is being tested, we never know if Id works... or is it we don't know if prettyUrlName works?

Also the original code is not "great" because it relies on reusing the first item from the expand mock. An unexpecting dev could change that mock and then break these tests. Or we could get rid of expand all together, and then these tests, still valid for search results, would break.

So it should have its own mocked up data that is for this test.

Reply:
@seyilonge-nci see my comment above...

Reply:
> @seyilonge-nci see my comment above...

👍

Reply:
The changes have been made to these tests! They now test both scenarios: with or without a pretty URL name. And are using mocked data from within the file.